bin/node/bench/src/tempdb.rs

```diff
@@ -32,10 +87,22 @@ impl TempDatabase {
 		TempDatabase(dir)
 	}
 
-	pub fn open(&mut self) -> Arc<dyn KeyValueDB> {
-		let db_cfg = DatabaseConfig::with_columns(1);
-		let db = Database::open(&db_cfg, &self.0.path().to_string_lossy()).expect("Database backend error");
-		Arc::new(db)
+	pub fn open(&mut self, db_type: DatabaseType) -> Arc<dyn KeyValueDB> {
+		match db_type {
+			DatabaseType::RocksDb => {
+				let db_cfg = DatabaseConfig::with_columns(1);
+				let db = Database::open(&db_cfg, &self.0.path().to_string_lossy()).expect("Database backend error");
+				Arc::new(db)
+			},
+			DatabaseType::ParityDb => {
+				Arc::new(ParityDbWrapper(
+					parity_db::Db::open(
+						&parity_db::Options::with_columns(self.0.path(), 1)
```

Comment:
Would be best to replicate these settings here:
https://github.com/paritytech/substrate/blob/master/client/db/src/parity_db.rs#L37-L42

Reply:
with this settings getting this, probably on reopen copied files
```
2020-04-29 17:05:30 Starting Trie read benchmark(empty database (200 keys), db_type: ParityDb)
2020-04-29 17:05:30 Started reindex for index 00_16
2020-04-29 17:05:30 Started reindex for index 00_17
2020-04-29 17:05:30 Eror replaying log: Corruption("Missing table"). Reverting
thread 'main' panicked at 'Failed to get key: db error: "Trie lookup error: Invalid state root: 0x978a92e555f0b502acfd735de980638138c37bbff2e3dfa66e7fe53384d3d541"', bin/node/bench/src/trie.rs:187:25
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
This is caused by lots of collisions that come from prefixed keys. So either prefixes should be stripped (as in #5769) or `uniform` option should not be used

Reply:
updated!